java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -1506,12 +1507,14 @@ public CompletableFuture<Long> bitop(
 
     @Override
     public CompletableFuture<Map<String, String[]>> lmpop(
-            @NonNull String[] keys, @NonNull PopDirection direction, long count) {
+            @NonNull String[] keys, @NonNull ListDirection listDirection, long count) {
```

Comment:
nit: not sure why you changed all these direction arguments.  
```suggestion
            @NonNull String[] keys, @NonNull ListDirection direction, long count) {
```

Reply:
IDE refactoring got to them, will change back